pocs-capstone/frontend/src/components/PetDisplay/PetDisplay.js

```diff
@@ -36,6 +36,36 @@ const HAPPY = 'H'
 const TASK_URL = "/tasks/"
 const USER_URL = "/user-data/"
 const TODAY = new Date()
+
+const WEIGHTS = [
+    '15 paperclips',
+    '3 nickels',
+    '5 pieces of bubblegum',
+    '4 mullberries',
+    '2/3 a medium radish',
+    '461 sequins',
+    '25 miniature toadstools',
+    '5 acorns',
+    '1 small egg',
+    '231.5 grains of rice',
+    '5 spoonfuls of cinnamon',
+    '272 apple seeds',
+    '1/17 of a grapefruit',
+    '8 cups of rose petals',
+    '2 tiny gray pebbles',
+    '12 jellybeans',
+    '1 strawberry',
+    '3 sheets of US letter sized printer paper',
+    '300 almond slivers',
+    '23/4 a slice of multigrain bread',
+    '1 big handful of cotton balls',
+    '4 clementine slices',
+    '150 wooden toothpicks',
+    '1 large thimble',
+    '3 #2 HB yellow Ticonderoga wooden pencils'
```

Comment:
``` 3 Ticonderoga #2 wooden pencils ```  maybe?

Reply:
lowkey I wanna go either "wooden pencils" or go the extremely overdescriptive route, but I already did that for the paper so I'm thinking I just make this wooden pencils